plugins_src/autouv/shaders/edge_filter.fs

```diff
@@ -8,43 +8,64 @@
 //  See the file "license.terms" for information on usage and redistribution
 //  of this file, and for a DISCLAIMER OF ALL WARRANTIES.
 //
-// Grabbed from tutorial By J�r�me Guinot jegx [at] ozone3d [dot] net
```

Comment:
Not utf-8?

Reply:
VSCode did that. In the status it says to be using UTF-8, LF, 4 spaces. So, I have not idea why it changed the header.

Reply:
It is Jermomes name that have changed, something is broken or was broken before.